datafusion/physical-plan/src/windows/bounded_window_agg_exec.rs

```diff
@@ -1553,7 +1552,7 @@ mod tests {
             Arc::new(BuiltInWindowExpr::new(
                 last_value_func,
                 &[],
-                LexOrderingRef::default(),
+                LexOrdering::default().as_ref(),
```

Comment:
I don't think the `as_ref()` is needed here  (it just makes another copy)

Reply:
using 

`&LexOrdering::default()`